### PR TITLE
Example command punctuation lacks consistency

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-5591
Link to docs preview:

https://57806--docspreview.netlify.app/openshift-rosa/latest/rosa_cli/rosa-manage-objects-cli.html
QE review:
- [ ] QE has approved this change.

<b>QE review N/A </b>
Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
/label peer-review-needed

Peer review squad: If you feel this ticket needs a QE review, please let me know. Based on the issue, I would not think that would be necessary, but would certainly welcome other opinions. Thanks!

**Review by mjpytlak — COMMENTED** [review]
@mletalie Updates are straight forward enough. Couple of requests. Please add a preview link `rosa` CLI reference topic, so I can verify the published output. Also - while you are here, all of the modules are missing a `:_content-type: <TYPE>` attribute. Please add them.

**Comment by mletalie** [comment]
Hi @mjpytlak, 
   Thanks for the heads up, completely spaced the preview link, it is there now. Will also add attributes ASAP. Appreciate it.

**Comment by mletalie** [comment]
Hello @mjpytlak,
    I have added the  `:_content-type: <TYPE>`  attribute to these modules. If you could take a look when you get a chance, I would appreciate it. Thanks.

**Comment by mletalie** [comment]
Hello @mjpytlak,
      I have added the `:_content-type: <TYPE>`attribute to the modules. Please take a look when you get a moment, thanks.

File: modules/rosa-common-commands.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 [id="rosa-common-commands_{context}"]
 = Common commands and arguments
-
+:_content-type: REFERENCE
```
Comment (mjpytlak) [inline modules/rosa-common-commands.adoc:8]: @mletalie Thanks for adding these. The content type attribute should sit before the anchor ID. [1]

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

**Comment by mletalie** [comment]
Hi @mjpytlak,
    Thanks for the wealth of information. Actually my first time dealing with adding content-type attributes to modules (or anywhere, for that matter), so I appreciate it!

**Comment by mjpytlak** [comment]
> Thanks for the wealth of information. Actually my first time dealing with adding content-type attributes to modules (or anywhere, for that matter), so I appreciate it!

Anytime.

**Review by bergerhoffer — COMMENTED** [review]
These CLI reference files don't look to be auto-generated, so this LGTM for merging

**Pull request merged**